### PR TITLE
Switch linter to Eslint

### PR DESCRIPTION
**Pull request by Luthaf**
And use https://prettier.io/ as code formatter.

@rosecers and @ceriottm please make sure to configure your editor to automatically format on save, this cuts on the commit noise. They have documentation on the prettier.io home page on how to do this!

### CODE REVIEW

**Comment by Luthaf** [comment]
Github Action was confused and picked the wrong commit for tests, hence the closing/reopening!

File: .github/workflows/website.yml [hunk 1]
```diff
@@ -10,59 +10,59 @@ jobs:
...
-        force_orphan: true
+      - name: deploy website to gh-pages
+        # only deploy website from master
+        if: github.ref == 'refs/heads/master'
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./gh-pages/
+          cname: chemiscope.org
+          force_orphan: true
```
Comment (rosecers) [inline .github/workflows/website.yml:56]: Are these new indentations a function of the new linter? Just asking because it's hard to parse.

File: Contributing.md [hunk 2]
```diff
@@ -3,24 +3,23 @@
...
 ## Area of contributions
 
 The first and best way to contribute to chemiscope is to use it and advertise it
 to other potential users. Other than that, you can help with:
 
-- documentation: correcting typos, making various documentation clearer;
-- bug fixes and improvement to existing code;
-- and many more …
+-   documentation: correcting typos, making various documentation clearer;
+-   bug fixes and improvement to existing code;
```
Comment (rosecers) [inline Contributing.md:15]:
```suggestion
-   bug fixes and improvements to existing code;
```

File: Contributing.md [hunk 3]
```diff
@@ -35,15 +34,15 @@ we can discuss it, and you have more chances to see your changes incorporated.
...
-- [ ] Create a Pull-Request;
-- [ ] Discuss your changes with the reviewers;
-- [ ] Have your code merged
-- [ ] Celebrate! :tada: :cake: :tada:
+-   [ ] Fork chemiscope;
+-   [ ] Create a local branch;
+-   [ ] Add code / correct typos / ...;
+-   [ ] Check that the code passes lint checks;
+-   [ ] Push to Github;
+-   [ ] Create a Pull-Request;
```
Comment (rosecers) [inline Contributing.md:42]:
```suggestion
-   [ ] Create a Pull Request;
```

File: Contributing.md [hunk 4]
```diff
@@ -79,15 +78,15 @@ Run tests and lints (we use `tslint` to ensure a consistent coding style):
 npm test
 ```
 
-Finally, you can push your code to Github, and create a [Pull-Request][PR] to
+Finally, you can push your code to Github, and create a [Pull-Request][pr] to
```
Comment (rosecers) [inline Contributing.md:81]:
```suggestion
Finally, you can push your code to Github, and create a [Pull Request][pr] to
```

File: .github/workflows/website.yml (hunk 1 above)
Comment (Luthaf) [reply .github/workflows/website.yml:56]: not the linter, prettier. We could disable yaml formatting with prettier, what do you think?
